ProjetSupaero2018/catkin_ws/src/display/worlds/PIE86.world

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
Please prune that file! i.e. remove the numerous non compulsory and/or useless lines.